app/controllers/ProjectEntryController.scala

```diff
@@ -1,23 +1,32 @@
 package controllers
 
-import javax.inject.Inject
-import models.{ProjectEntry, ProjectEntryRow, ProjectEntrySerializer, StorageEntryRow}
-import play.api.Configuration
+import javax.inject.{Inject, Singleton}
+
+import helpers.ProjectCreateHelper
+import models._
+import play.api.{Configuration, Logger}
 import play.api.db.slick.DatabaseConfigProvider
 import play.api.libs.json.JsValue
-import play.api.mvc.Request
+import play.api.mvc.{Action, BodyParsers, Controller, Request}
 import slick.driver.JdbcProfile
 import slick.lifted.TableQuery
 import slick.driver.PostgresDriver.api._
+import play.api.libs.json.{JsError, Json}
+
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.util.{Failure, Success}
 
 /**
   * Created by localhome on 17/01/2017.
   */
-class ProjectEntryController @Inject() (config: Configuration, dbConfigProvider: DatabaseConfigProvider)
-  extends GenericDatabaseObjectController[ProjectEntry] with ProjectEntrySerializer
+@Singleton
+class ProjectEntryController @Inject() (config: Configuration, dbConfigProvider: DatabaseConfigProvider, projectHelper:ProjectCreateHelper)
+  extends Controller with ProjectEntrySerializer with ProjectRequestSerializer
```

Comment:
Removing the generic CRUD views for the time being as create/delete are not applicable (this object requires significantly more work that simple db insert/delete)